frontend/src/components/organism/Projections.tsx

```diff
@@ -61,7 +61,18 @@ const Projections: React.FC<{ className?: string }> = ({ className }) => {
               )) ? (
               <h3>{date.format('MMMM YYYY')}</h3>
             ) : null}
-            <div className="projection-row">
+            <div
+              className={
+                t.type === 'IN'
+                  ? 'projection-row projection-row-in'
+                  : 'projection-row projection-row-ex'
+              }
+              style={
```

Comment:
Avoid inline-styling, this is part of the reason why we use `styled-components`.

This can be solved in two ways. Either, keep the `className`, and use a sibling selector;
```css
.projection-row-ex + .projection-row-in,
.projection-row-in + .projection-row-ex {
  margin-top: 0.3em;
}
```

_or_ what I think would be the cleanest solution; move this entire object out into a new Styled Component;

```tsx
interface IRowEntryProps {
  readonly type: ITransaction['type'];
  readonly gapAbove: boolean;
}

const RowEntry = styled.div<IRowEntryProps>`
  /* styling from original .projection-row goes here */

  background-color: ${props => props.type === "IN" ? "#456" : "#123" };
  ${props => props.gapAbove && "margin-top: 0.3em;" }
`;
```

With the latter solution, you move the styling out of the rendering of these objects, and declutter the code a bit.

You would then use it like this;

```tsx
<RowEntry type={tx.type} gapAbove={blabla === blabla}>
  {/* content */}
</RowEntry>
```

More about this at https://www.styled-components.com/docs/api#using-custom-props.

Wrote this without testing, so might have to make some smaller modifications to the code.